x-pack/package.json

```diff
@@ -23,7 +23,7 @@
     }
   },
   "resolutions": {
-    "**/@types/node": "10.12.27"
+    "**/@types/node": "~10.17.17"
```

Comment:
What's the point of using `~` here? I personally think resolutions should specify a specific version, but also if `10.19.0` was released then we would want to upgrade I think so `^` seems more appropriate if we're going to use a range.

Reply:
Good point. I wanted to make sure we didn't go above our pinned Node.js version but still allow patch releases - so technically I wanted to do `~10.19.0`, but since v10.19 didn't exist, I went with the highest one below that which did.

But I was actually just about to update the range to something like this instead: `>=10.17.17 <10.20.0`

What do you think about that?

Reply:
Works for me too, though because of yarn.lock we can't upgrade without another PR anyway so just using `"10.17.17"` works for me too.

Reply:
By making it a range, it makes it easier to update the version in the future by just deleting the `@types/node` entry in `yarn.lock` and running `yarn kbh bootstrap` which will re-write the entry in `yarn.lock` to the latest version within the accepted range. So I just thought I would make it a little less hassle to keep up-to-date this way 😃